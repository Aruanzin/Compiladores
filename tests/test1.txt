var 1xa;
BEGIN
  x := 9.5;
END.
