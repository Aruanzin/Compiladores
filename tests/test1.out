VAR, VAR
x, ident
;, simbolo_ponto_virgula
BEGIN, BEGIN
x, ident
:=, simbolo_atribuicao
42, numero
;, simbolo_ponto_virgula
END, END
., simbolo_ponto
